docs/earth-command/earth-command.md

```diff
@@ -144,6 +138,8 @@ Sets the git username to use for non-SSH git authentication. For more informatio
 
 To prevent the need to specify this option on every `earth` invocation, it is recommended to specify the env var form in a file like `.bashrc` or `.profile`.
 
+This option is now deprecated. Please use the [configuration file](../earth-config/earth-config.md) instead.
+
```

Comment:
Did you mean to remove this part too:

```
Sets the git username to use for non-SSH git authentication. For more information see the [Authentication page](../guides/auth.md).

To prevent the need to specify this option on every `earth` invocation, it is recommended to specify the env var form in a file like `.bashrc` or `.profile`.
```

Similar for others too.

Reply:
do you think it's better to just cut it down to this?

```
##### `--git-username <git-user>`

Also available as an env var setting: `GIT_USERNAME=<git-user>`.

This option is now deprecated. Please use the [configuration file](../earth-config/earth-config.md) instead.

```

Reply:
Yes exactly - that's what I meant. We should do this for all the deprecated options.